.github/pull_request_template.md

```diff
@@ -0,0 +1,45 @@
+<!--
+Thank you for submitting a PR to the rust rpm implementation!
+-->
+
+# PR Content Desc
+
+<!---
+DELETE all that do not apply:
+-->
+
+- 🩹 Bug Fix
+- 🦚 Feature
+- 📙 Documentation
+- 🦣 Legacy
+- 🪣 Misc
+
+<!---
+Mention the linked issue here.
+This will automatically close the issue once the PR is merged
+and creates a cross reference.
```

Comment:
What if there's not a linked issue?

Reply:
Well, then one should be created, or not linked. I don't think it should contain any possible path of the decision tree, I will apply suggestions to change it though, no strong opinion.

Reply:
(I have no strong opinion either, just wanted to raise the point)